src/components/LeaderboardList.js

```diff
@@ -61,12 +61,12 @@ class LeaderboardList extends InjectedComponent<InjectedProps, Props> {
     index: number,
   }): React.Element<any> => (
     <ListItem
-      avatar={<UserAvatar userName={item.userName} />}
+      avatar={<UserAvatar userName={item.userName || ''} />}
       item={item}
-      onPress={this._onListItemPress}
+      onPress={item.userID ? this._onListItemPress : undefined}
       rightIcon={<PintCounter ounces={item.totalOunces} />}
       subtitle={`${item.totalOunces.toFixed(1)} oz`}
-      title={`${index + 1}. ${item.userName}`}
+      title={`${index + 1}. ${item.userName || 'anonymous'}`}
```

Comment:
I actually do not want anonymous to show up here. 'anonymous' would mean that it was done with anonymous pours.

I think it makes more sense to keep this empty. I'm pretty sure we don't write `anonymous` for pours you don't have access to.

Reply:
ok